Gruntfile.js

```diff
@@ -71,7 +71,7 @@ module.exports = function (grunt) {
             },
             "node-mac": {
                 "dest"      : "<%= downloads %>",
-                "src"       : "http://nodejs.org/dist/v<%= node.version %>/node-v<%= node.version %>-darwin-x86.tar.gz"
+                "src"       : "http://nodejs.org/dist/v<%= node.version %>/node-v<%= node.version %>-darwin-x64.tar.gz"
```

Comment:
Yeah. Now that we have moved to Chokidar, I don't think we will have any references to native node extensions. I was referring to this library which we were using earlier for file watching.

https://github.com/adobe/brackets/tree/release/src/filesystem/impls/appshell/node/node_modules/fsevents

I think then it should be fine to go with 64 bit.